Replace the use of XMLPrettyPrint from PyXML in xm with stdlib functionality.
This was reported by Toshio Ernie Kuratomi at
https://bugzilla.redhat.com/show_bug.cgi?id=842843

Signed-off-by: Michael Young <m.a.young@durham.ac.uk>
--- xen-4.2.0/tools/python/xen/xm/create.py.orig	2012-05-12 16:40:48.000000000 +0100
+++ xen-4.2.0/tools/python/xen/xm/create.py	2012-08-12 17:59:56.464270707 +0100
@@ -1543,8 +1543,7 @@
             SXPPrettyPrint.prettyprint(config)
 
         if opts.vals.xmldryrun and serverType == SERVER_XEN_API:
-            from xml.dom.ext import PrettyPrint as XMLPrettyPrint
-            XMLPrettyPrint(doc)
+            print doc.toprettyxml()
 
     if opts.vals.dryrun or opts.vals.xmldryrun:
         return                                               
